app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module TwoFactorAuthCode
+  class PhoneDeliveryPresenter < TwoFactorAuthCode::GenericDeliveryPresenter
+    include TwoFactorAuthCode::Totpable
+    include TwoFactorAuthCode::Phoneable
```

Comment:
They are definitely coupled to these presenters, but I kept them as separate modules for a couple of reasons:

* Not all the presenters will use all of these methods.
* Future presenters also need these modules. One of them, the OTP presenter ( in #902 ) also uses the methods on `phonable`. Additionally, a presenter for the recovery code screen would probably need to use the `totpable` module.

If you think it makes more sense to just put these methods inside the generic presenter class I will go ahead an do that!